src/main/scala/it/pagopa/interop/tenantprocess/service/impl/TenantManagementServiceImpl.scala

```diff
@@ -135,4 +135,28 @@ final case class TenantManagementServiceImpl(
     ec: ExecutionContext,
     readModel: ReadModelService
   ): Future[PaginatedResult[PersistentTenant]] = ReadModelTenantQueries.listConsumers(name, producerId, offset, limit)
+
+  override def addTenantMail(tenantId: UUID, mailSeed: MailSeed)(implicit
+    contexts: Seq[(String, String)]
+  ): Future[Unit] = withHeaders { (bearerToken, correlationId) =>
+    val request = tenantApi.addTenantMail(xCorrelationId = correlationId, tenantId = tenantId, mailSeed = mailSeed)(
+      BearerToken(bearerToken)
+    )
+    invoker.invoke(request, s"Adding mail ${mailSeed.address} to tenant $tenantId").recoverWith {
```

Comment:
Potential sensitive info
```suggestion
    invoker.invoke(request, s"Adding mail with id ${mailSeed.id} to tenant $tenantId").recoverWith {
```